gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/QueryRequestContext.java

```diff
@@ -42,4 +43,21 @@ public void mapAliasToTimeAggregation(String alias, TimeAggregation timeAggregat
   public TimeAggregation getTimeAggregationByAlias(String alias) {
     return aliasToTimeAggregation.get(alias);
   }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
Was this being used in any maps?

Reply:
I'm checking for equality in unit tests.